data/blog/web-preferences-fix.md

```diff
@@ -0,0 +1,66 @@
+---
+title:  WebPreferences Vulnerability Fix
+author: ckerr
+date: '2018-08-22'
+---
+
+A remote code execution vulnerability has been discovered affecting apps with the ability to open nested child windows on Electron versions (3.0.0-beta.6, 2.0.7, 1.8.7, and 1.7.15). This vulnerability has been assigned the CVE identifier [CVE-2018-15685].
+
+---
+
+## Affected Platforms
+
+You are impacted if:
+
+1. You embed _any_ remote user content, even in a sandbox
+2. You accept user input with any XSS vulnerabilities
+
+_Details_
+
+You are impacted if any user code runs inside an `iframe` / can create an `iframe`. Given the possibility of an XSS it can be assumed that most apps are vulnerable to this case.
+
+You are also impacted if you open any of your windows with the `nativeWindowOpen: true` or `sandbox: true` option.  Although this vulnerability also requires an XSS to exist in your app, you should still apply one of the mitigations below if you use either of these options.
```

Comment:
"an XSS" is not really a thing. Maybe "an XSS vulnerability" or "an XSS susceptibility" or similar.